### PR TITLE
fix(cas-95): member list shows duplicated addresses

### PR DESCRIPTION
**Pull request by germanurrus**
Description: 
On the community page, the member list shows duplicated addresses:
Screen Shot 2022-06-08 at 12 28 50

The endpoint used to retrieve data is `communities/{communityId}/users` the issue is that payload is arranged this way:
Screen Shot 2022-06-08 at 12 29 25
user has more than one role there will be as many entries by address as user types that address has. 
One possible solution would be to group data in the frontend, but since results are being paginated grouping needs to be done on every request and existing data in frontend state need to be used to group by address. 
In addition this endpoint is used to count community members, but since a user could have many roles, result may change depending on user roles per address. 

This PR changes the result payload for the endpoint `communities/{communityId}/users` to this:

Screen Shot 2022-06-09 at 10 59 25

also table has been updated to hide the last two columns:
Screen Shot 2022-06-10 at 13 56 55

### CODE REVIEW

File: backend/main/models/community_users.go [hunk 1]
```diff
@@ -42,12 +64,12 @@ func GetUsersForCommunity(db *s.Database, communityId, start, count int) ([]Comm
...
+		return []CommunityUserType{}, 0, nil
 	}
 
 	// Get total number of users
 	var totalRecords int
-	countSql := `SELECT COUNT(*) FROM community_users WHERE community_id = $1`
+	countSql := `SELECT COUNT(*) FROM (SELECT addr FROM community_users WHERE community_id = $1 group BY community_users.addr) as temp_users_addr`
 	_ = db.Conn.QueryRow(db.Context, countSql, communityId).Scan(&totalRecords)
 
 	return users, totalRecords, nil
```
Comment (germanurrus) [inline backend/main/models/community_users.go:75]: This function is used here: https://github.com/DapperCollectives/CAST/blob/958b9eea7f77c01aa3c3378da7df373d0d843da6/backend/main/server/app.go#L1498-L1499
the endpoint allows filtering, that with change will have two types of return payload. They question is if this should be implemented on a separate endpoint.

**Comment by germanurrus** [comment]
@CJamesKeller this endpoint result is also affecting the members count on the community new header

**Comment by germanurrus** [comment]
I pushed to style changes to hide `voting streak` and `score`

File: backend/main/models/community_users.go (hunk 1 above)
Comment (jacksonConrad) [reply backend/main/models/community_users.go:75]: i think it probably should be a separate endpoint, but we can merge this in as-is and i'll create a backlog ticket to refactor.

File: backend/main/models/community_users.go [hunk 2]
```diff
@@ -29,11 +37,25 @@ type CommunityUserPayload struct {
...
 }
 
-func GetUsersForCommunity(db *s.Database, communityId, start, count int) ([]CommunityUser, int, error) {
-	var users = []CommunityUser{}
+func GetUsersForCommunity(db *s.Database, communityId, start, count int) ([]CommunityUserType, int, error) {
+	var users = []CommunityUserType{}
 	err := pgxscan.Select(db.Context, db.Conn, &users,
 		`
-		SELECT * FROM community_users WHERE community_id = $1
+		SELECT
```
Comment (jacksonConrad) [inline backend/main/models/community_users.go:44]: there might be a more efficient way to write this query but i'm not totally sure off the top of my head.

we can revisit later if its a problem. :)

**Review by jacksonConrad — APPROVED** [review]

File: backend/main/models/community_users.go (hunk 2 above)
Comment (germanurrus) [reply backend/main/models/community_users.go:44]: yeah I agree, we could have functions running each of the CASE statements to simply the query and maybe creating a view or a materialized view could help too.

**Pull request merged**